src/views/main.py

```diff
@@ -159,5 +176,5 @@ def get_data(group_by):
     index_column = "_id"
     collection = "variant"
     results = DataTablesServer(request, columns, index_column, collection,
-                               group_by).output_result_on_queried_fields()
+                            group_by).output_result_on_queried_fields()
```

Comment:
Is this change autocorrected? 
Seems like previous/old indentation was 'correct' based on the vertical alignment with line 178 
`results = DataTablesServer(request, columns, index_column, collection,`

Reply:
no particular reason.
I will change this in a seperate branch, that I will use to clean-up and re-format all the code, to comply to our ([link](https://dev.azure.com/umcu/LAB-Bioinformatics-Genetics/_wiki/wikis/Bioinformatics-Genetics.wiki/1276/Clear-code ) and the PEP8 Style guide.